users/forms.py

```diff
@@ -36,10 +35,15 @@
 class CodeSubmitForm(forms.Form):
     SUBMIT_PATH = config_info.get_config('path', 'submission_code_path')
     LANGUAGE_CHOICE = tuple(config_info.get_config_items('compiler_option'))
+    BACKEND_VERSION = config_info.get_config('system_version', 'backend')
+    GCC_VERSION = config_info.get_config('system_version', 'gcc')
+    GPP_VERSION = config_info.get_config('system_version', 'gpp')
 
-    pid = forms.CharField()
+    pid = forms.CharField(label='Problem ID')
     language = forms.ChoiceField(choices=LANGUAGE_CHOICE, initial=Submission.CPP,
-                                 widget=forms.RadioSelect())
+                                 widget=forms.RadioSelect(),
+                                 help_text="Backend: %s<br>gcc: %s, g++: %s"
```

Comment:
The ``Ubuntu`` seem to appear many times, will we eliminate the version to ``4.7.3``?